Task_1/Task_1.cpp

```diff
@@ -1,6 +1,118 @@
 ﻿#include <iostream>
 
+using namespace std;
+
+//ЧИСЛО СУДЬБЫ. Цифры полной даты рождения суммируются, пока не получится одна цифра. Разработайте функцию и вычислите свое число судьбы)
+
+int enter(int number, string str);
+bool check(int y, int m, int d);
+int numberOfFate(int y, int m, int d);
+
 int main()
 {
-    std::cout << "Hello World!\n";
+    int year = 0, month = 0, day = 0;
+
+    cout << "Enter your birthday: " << endl;
+    year = enter(year, "Year");
+    month = enter(month, "Month");
+    day = enter(day, "Day");
+
+    if (check(year, month, day) != true)
+    {
+        cout << "\nThere is no such date." << endl;
+
+        return -1;
+    }
+    else {}
+
+    cout << "\nNumber of fate: " << numberOfFate(year, month, day) << endl;
```

Comment:
the same

Reply:
![image](https://user-images.githubusercontent.com/71211228/111910684-755f8b00-8a73-11eb-807c-50b496f10f47.png)